### PR TITLE
Sprint 46

### PR DESCRIPTION
**Pull request by ericsorenson**
# Sprint 46

- #4881 | Index number - do not give an item an index number until it’s been served
- #5666 | Development Team: Allow production domain configuration
- #4533 | Ability to easily reference case details while generating an order and QCing
- #5841 | Revisit flow: Proposed/Signed Stipulated Decision
- #5511 | Messaging Re-design clean-up - Remove old messaging flow

### CODE REVIEW

File: shared/src/business/utilities/shouldGenerateDocketRecordIndex.test.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    };
+    const result = shouldGenerateDocketRecordIndex({
+      caseDetail,
+      docketRecordEntry,
+    });
+
+    expect(result).toEqual(true);
+  });
+
+  it('returns true for initial document types filed after the petition if the document is being served', () => {
```
Comment (julialeague) [inline shared/src/business/utilities/shouldGenerateDocketRecordIndex.test.js:142]: @ericsorenson The expectation is `false` on line 180 - is this test description inaccurate?

**Comment by julialeague** [comment]
- [x] begin by deleting any existing copy of the Git repository, running `docker system prune` to remove any cached artifacts from any prior code review
- [x] clone a fresh copy of the Git repository (`git clone https://github.com/ustaxcourt/ef-cms.git`), fetch the pull request and name it for the sprint (e.g., `git fetch origin pull/347/head:sprint-46`), and then switch to that branch (e.g. `git checkout sprint-46`)
- [x] review the pull request itself, to get oriented
	- [x] read the description of the pull request, which should summarize the changes made
	- [x] read through every task on the Scrum board that's encompassed by this pull request
	- [x] read the description of the commits that comprise the pull request
- [x] stand up the site locally, with `./docker-run.sh`
	- [x] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
		- [x] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
		- [x] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/), including Internet Explorer 11
		- [x] test in Mobile Safari and Mobile Chrome, with the caveat that not all internal Court functionality will be necessary on these platforms
	- [x] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [x] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [x] review for accessibility
		- [x] use an automated audit tool, such as Chrome Audit or aXe
		- [x] navigate site only with the keyboard
		- [x] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [x] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [x] [review static code analysis results](https://sonarcloud.io/organizations/flexion-github/projects) in the vendor’s SonarCloud environment
- [x] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [x] skim all new code, in the context of existing code, [looking for problems](#what-we-look-for) (knowing that the vast majority of new code will be covered by tests)
- [x] review all tests
	- [x] look at code coverage of tests in [the vendor's SonarCloud environment](https://sonarcloud.io/organizations/flexion-github/projects)
	- [x] methodically review all new tests for correctness, quality of naming
- [x] determine what code isn’t tested, review that rigorously
- [x] review documentation to ensure that it matches changes
- [x] provide comments on the pull request on GitHub, as necessary
	- [x] for comments that are specific to a particular line of code, comment on those specific lines
	- [x] for comments that are more general, attach the comment to a random line in `README.md` (as opposed to commenting on the pull request itself), to be able to use GitHub's ability to thread discussions on those comments
- [x] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

File: shared/src/business/utilities/shouldGenerateDocketRecordIndex.test.js (hunk 1 above)
Comment (ericsorenson) [reply shared/src/business/utilities/shouldGenerateDocketRecordIndex.test.js:142]: Updated!

**Comment by julialeague** [comment]
@mmarcotte, I recommend you accept this pull request into the staging environment. Thanks!

**Review by julialeague — APPROVED** [review]

**Pull request merged**